packages/crayons-core/src/components/icon/icon-utils.ts

```diff
@@ -0,0 +1,60 @@
+// Adding Intersection-Observer : fw-icon
+export const waitUntilVisible = (
+  intersectionObserver: IntersectionObserver,
+  xRootMargin: string,
+  el: HTMLElement,
+  callback: () => void
+): void => {
+  intersectionObserver = new IntersectionObserver(
+    (entries) => {
+      entries.forEach((entry) => {
+        if (entry.isIntersecting) {
+          intersectionObserver.disconnect();
+          intersectionObserver = undefined;
+          callback();
+        }
+      });
+    },
+    { rootMargin: xRootMargin }
+  );
+
+  intersectionObserver.observe(el);
+};
+
+// Icon fetch-api with memoization :  fw-icon
+const CRAYONS_ICONS_ASSET_PATH =
```

Comment:
At present we are deploying to `canary` tag. The above url will not work with the current release setup.

Reply:
Fixed